File: DSA/temp.py
projects = [
    {
        "name":"firstProject",
        "tags":[],
        "files":[]
    },
    {
        "name":"secondProject",
        "tags":[],
        "files":['UserFile/Hello']
    },
    {
        "name":"thirdProject",
        "tags":[],
        "files":[]
    } 
]

for project in projects:
    if project["name"] == "firstProject":
        if "UserFile/Hello" in project["files"]: print("fileExists")
        else: 
            project["files"].append("UserFile/Help")
            print(projects)
            break
        
print(projects)

File: DSA/temp.js
projects = [
    {
        "name": "firstProject",
        "tags": [],
        "files": []
    },
    {
        "name": "secondProject",
        "tags": [],
        "files": ['UserFile/Hello', 'UserFile/Hello', 'UserFile/Hello', 'UserFile/Hello']
    },
    {
        "name": "thirdProject",
        "tags": [],
        "files": []
    }
]

projectFiles = projects.filter(project => project.name === "secondProject")[0].files
projectFiles.map((files) => console.log(files))